packages/shared/src/components/fields/MarkdownInput/index.tsx

```diff
@@ -72,6 +97,8 @@ function MarkdownInput({
     enabledCommand,
   });
 
+  useImperativeHandle(ref, () => ({ textareaRef, onMentionCommand }));
```

Comment:
Just to check if we need this or we use this handle on ref anywhere in the code? Can not see it.

Reply:
We use this on the Footer button which is located outside the editor. I will line the line of code once I stumble upon it.

Reply:
Should be from this line:

https://github.com/dailydotdev/apps/pull/1869/files#diff-eba809a0ead9105aaba7c14726f5ed2bc7c5b59485163dd6b6d9f9de7d8fef9bR104